lib/measures/tbd/resources/psi.rb

```diff
@@ -199,15 +216,7 @@ def processTBDinputs(surfaces, edges, set, io_f, schema_f)
       end
 
       p = io[:unit].first[:psi]
-      raise "Unit #{p} mismatch?"     unless psi.set.has_key?(p)
-      raise "Unit #{p} rimjoist?"     unless psi.set[p].has_key?(:rimjoist)
-      raise "Unit #{p} parapet?"      unless psi.set[p].has_key?(:parapet)
-      raise "Unit #{p} fenestration?" unless psi.set[p].has_key?(:fenestration)
-      raise "Unit #{p} concave?"      unless psi.set[p].has_key?(:concave)
-      raise "Unit #{p} convex?"       unless psi.set[p].has_key?(:convex)
-      raise "Unit #{p} balcony?"      unless psi.set[p].has_key?(:balcony)
-      raise "Unit #{p} party?"        unless psi.set[p].has_key?(:party)
-      raise "Unit #{p} grade?"        unless psi.set[p].has_key?(:grade)
+      psi.complete?(p)
```

Comment:
Here you would do `raise 'Not Complete' unless psi.complete?(p)`

Reply:
Got it.